golang/inbound.go

```diff
@@ -91,7 +90,12 @@ func (c *Connection) handleCallReq(frame *Frame) bool {
 	response.messageForFragment = func(initial bool) message {
 		if initial {
 			call.AddAnnotation(AnnotationKeyServerSend)
-			call.Report(callReq.Tracing, c.traceReporter)
+			targetEndpoint := TargetEndpoint{
```

Comment:
Not sure what you mean. You probably need to give me a code walkthrough. Otherwise I will have no idea about those things.